Python-Data-Generation-Scripts/sql-query.json

```diff
@@ -1,44 +1,36 @@
 {
-    "connection": {
-        "host": "localhost",
-        "user": "root",
-        "password": "yourpassword",
-        "database": "hospital"
-    },
```

Comment:
Move this segment to `connection.json`

`connection.json` should look like this
```json
{
    "host": "localhost",
    "user": "root",
    "password": "p@$$W0rD",
    "database": "hospital"
}
```